code/modules/economy/price_list.dm

```diff
@@ -967,17 +969,19 @@
 	for(var/obj/item/ammo_casing/i in stored_ammo)
 		. += i.get_item_cost(export)
 
-/obj/item/ammo_casing/price_tag = 0
+/obj/item/ammo_casing/price_tag = 0.2
```

Comment:
On my way to export ammo casings left after a gunfight for 20 credits per a 100 stack.

Reply:
I don't know why but I believe someone would do this unironically now.